src/private/remoteCamera.ts

```diff
@@ -0,0 +1,305 @@
+   */
+  export interface SessionStatus {
+    /**
+     * Whether the far-end user is controlling a  device.
+     */
+    inControl: boolean;
+    /**
+     * Reason the  control session was terminated.
+     */
+    terminatedReason?: SessionTerminatedReason;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate the reason the session was terminated.
+   */
+  export enum SessionTerminatedReason {
+    None,
+    ControlDenied,
+    ControlNoResponse,
+    ControlBusy,
+    AckTimeout,
+    ControlTerminated,
+    ControllerTerminated,
+    DataChannelError,
+    ControllerCancelled,
+    ControlDisabled,
+  }
+
+  GlobalVars.handlers['remoteCamera.capableParticipantsChange'] = handleCapableParticipantsChange;
+  GlobalVars.handlers['remoteCamera.handlerError'] = handleHandlerError;
+  GlobalVars.handlers['remoteCamera.deviceStateChange'] = handleDeviceStateChange;
+  GlobalVars.handlers['remoteCamera.sessionStatusChange'] = handleSessionStatusChange;
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Fetch a list of the participants with controllable-cameras in a meeting.
+   * @param callback Callback contains 2 parameters, error and participants.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * participants can either contain an array of Participant objects, incase of a successful fetch or null when it fails
+   * participants: object that contains an array of participants with controllable-cameras
+   */
+  export function getCapableParticipants(
+    callback: (error: SdkError | null, participants: Participant[] | null) => void,
+  ): void {
+    if (!callback) {
+      throw new Error('[remoteCamera.getCapableParticipants] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('remoteCamera.getCapableParticipants');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Request control of a participant's camera.
+   * @param participant Participant specifies the participant to send the request for camera control.
+   * @param callback Callback contains 2 parameters, error and requestResponse.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * requestResponse can either contain the true/false value, incase of a successful request or null when it fails
+   * requestResponse: True means request was accepted and false means request was denied
+   */
+  export function requestControl(
+    participant: Participant,
+    callback: (error: SdkError | null, requestResponse: boolean | null) => void,
+  ): void {
+    if (!participant) {
+      throw new Error('[remoteCamera.requestControl] Participant cannot be null');
+    }
+    if (!callback) {
+      throw new Error('[remoteCamera.requestControl] Callback cannot be null');
+    }
+    ensureInitialized();
```

Comment:
It's probably best to pass `FrameContexts.sidePanel` to the `ensureInitialized();` call since these APIs won't work from any other context.